README.md

```diff
@@ -131,7 +133,7 @@ app.use(errorHandler({ log: false }));
   "final:after": {
     "strong-error-handler": {
       "params": {
-        "debug": true,
+        "debug": false,
```

Comment:
This should stay `true` - you want to see detailed error logs when running in development mode.